src/drop.ts

```diff
@@ -15,7 +16,10 @@ import { purry } from "./purry";
  * @lazy
  * @category Array
  */
-export function drop<T>(array: ReadonlyArray<T>, n: number): Array<T>;
+export function drop<const T extends Array<unknown>, N extends number>(
```

Comment:
We currently don't use `const` in our functions, and for consistency we shouldn't here either.
```suggestion
export function drop<T extends Array<unknown>, N extends number>(
```

Reply:
it means we lose some of the typings (will be typed as `[number, number]` instead of `[1, 2]`)

but sure, let's go for it, for the sake of consistency 😄